app/components/AdminDropdown/ProjectTypeDropdown.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { AgencyBudget, createAgencyBudget } from "~/gen";
+import { ProjectTypeDropdown } from "./ProjectTypeDropdown";
+import { render, screen } from "@testing-library/react";
+import { userEvent } from "@testing-library/user-event";
+import { act } from "react";
+
+describe("ProjectTypeDropdown", () => {
+  let agencyBudgets: Array<AgencyBudget> = [];
+  beforeAll(() => {
+    agencyBudgets = Array.from(Array(1), () => createAgencyBudget());
+  });
+
+  it("should render project type form details and options", () => {
+    const onSelectValueChange = vi.fn();
+    render(
+      <ProjectTypeDropdown
+        onSelectValueChange={onSelectValueChange}
+        projectTypes={agencyBudgets}
+      />,
+    );
+    expect(screen.getByLabelText("Project Type")).toBeInTheDocument();
+    const firstAgencyBudgetLabel = `${agencyBudgets[0].type}`;
+    expect(screen.getByText(firstAgencyBudgetLabel)).toBeInTheDocument();
+  });
+
+  it("should set search params when changing the project type", async () => {
```

Comment:
Can you change the text describing this test? I think it's a holdover from earlier versions of the code where the query params updated immediately. The component shouldn't update the search params, it should just call it's `onSelectValueChange` with the new value.